repositories/UserRepository.ts

```diff
@@ -91,4 +91,10 @@ export class UserRepository extends BaseRepository<UserModel> {
       })
       .execute();
   }
+
+  public async getAllPrivateProfiles() {
+    return this.repository.find({
+      select: ['uuid', 'email', 'handle', 'accessType'],
+    });
```

Comment:
@dowhep I know we wanted to potentially return PrivateProfile for the `GET` route for users with access types, but is there a way to directly obtain those fields without manually selecting them like this? I noticed the `findAll()` method at the top of this file, but it seems like it returns a full `UserModel`.

On the other hand, could this be sufficient (do we need to pull more fields than these)?

Reply:
If you want to call this "getAllPrivateProfiles" you should use the UserModel.getFullUserProfile() for each of the users.

Reply:
The current implementation I believe invalidates the interface PrivateProfile that you promised in getAllUsersWithAccessLevels()